packages/cli-repl/src/cli-repl.ts

```diff
@@ -1,74 +1,57 @@
-/* eslint no-console: 0 */
-
 import { CliServiceProvider, NodeOptions, CliOptions } from '@mongosh/service-provider-server';
-import { ShellInternalState } from '@mongosh/shell-api';
-import { ShellEvaluator, ShellResult } from '@mongosh/shell-evaluator';
-import formatOutput, { formatError } from './format-output';
-import { LineByLineInput } from './line-by-line-input';
-import { TELEMETRY_GREETING_MESSAGE, MONGOSH_WIKI } from './constants';
 import { ConfigManager, ShellHomeDirectory } from './config-directory';
 import { MongoshInternalError, MongoshWarning } from '@mongosh/errors';
-import { changeHistory, retractPassword } from '@mongosh/history';
-import completer from '@mongosh/autocomplete';
+import { retractPassword } from '@mongosh/history';
```

Comment:
isn't this supposed to be `redactPassword`? I think this may be some kind of typo.

Reply:
Yeah, uh … this seems like a typo :smile:

https://github.com/mongodb-js/mongosh/blob/4de0838bcd27207a8a149ace6811d5cb83c9a6e4/packages/history/src/index.ts#L2-L3